infra.bs

```diff
@@ -258,11 +262,31 @@ inclusive.
 
 <h3 id=strings>Strings</h3>
 
-<p>A <dfn export>string</dfn> is a sequence of <a>code points</a>. Strings are denoted by double
-quotes and monospace font.
+<p>A <dfn export>JavaScript string</dfn> is a sequence of unsigned 16-bit integers, also known as
+<dfn export lt="code unit">code units</dfn>. A <a>JavaScript string</a> can also be interpreted as
```

Comment:
It isn't UTF-16. It's what Unicode (TUS Section 2.7) calls a _Unicode 16-bit string_. It's worth glancing at that section of Unicode